desktop/ui/src/main/java/org/datacleaner/widgets/tabs/VerticalTabbedPane.java

```diff
@@ -56,8 +56,7 @@
 
     private static final Color COLOR_SELECTED_FOREGROUND = WidgetUtils.BG_COLOR_BLUE_DARK;
     private static final Color COLOR_SELECTED_BACKGROUND = WidgetUtils.BG_COLOR_LESS_BRIGHT;
-    private static final Border BORDER_TABS = new CompoundBorder(WidgetUtils.BORDER_LIST_ITEM_SUBTLE, new EmptyBorder(
-            10, 4, 10, 4));
+    private static final Border BORDER_TABS = new CompoundBorder(WidgetUtils.BORDER_LIST_ITEM_SUBTLE, new EmptyBorder(10, 4, 10, 4));
```

Comment:
I see quite a lot of these diffs caused by different formatter settings. I think we normally use a line width of 140 chars - how wide are your settings?